olp-cpp-sdk-core/CMakeLists.txt

```diff
@@ -399,6 +401,7 @@ target_link_libraries(${PROJECT_NAME}
         ${CMAKE_THREAD_LIBS_INIT}
     PRIVATE
         leveldb::leveldb
+        stdc++fs
```

Comment:
We are C++11 only. Use boost::filesystem if it is needed.

Reply:
boost filesystem is also not available sadly

Reply:
Then please use Linux native function in test file only and add wrap the test in #if _linux_.